src/test/java/org/folio/processing/mapping/mapper/writer/marc/MarcBibRecordModifierTest.java

```diff
@@ -156,11 +157,12 @@ public void shouldRemoveUnmatchedSubfield0OnSeveralSubfield0WhenMatchedIsLastOne
     testMarcUpdating(incomingParsedContent, expectedParsedContent, 1);
   }
 
+
   @Test
   public void shouldRemoveLinksOnSubfield0Change() throws IOException {
     // given
     var incomingParsedContent = "{\"leader\":\"00049nam  22000371a 4500\",\"fields\":[{\"001\":\"ybp7406411\"}," +
-      "{\"100\":{\"subfields\":[{\"a\":\"electronic updated\"},{\"0\":\"test1\"},{\"9\":\"bdbf59b7-913b-42ac-b1c6-e50ae7b00e6a\"}],\"ind1\": \" \",\"ind2\":\" \"}}," +
+      "{\"100\":{\"subfields\":[{\"a\":\"electronic updated\"},{\"0\":\"test1\"}],\"ind1\": \" \",\"ind2\":\" \"}}," +
```

Comment:
Subfield $9 was added to incoming in order to check it's not saved when link removed